app/controllers/rubcop_test_controller.rb

```diff
@@ -0,0 +1,17 @@
+class RubocopController  < ApplicationController
+	def initialize
```

Comment:
__[development][rubocop]__
* Style/IndentationWidth: Use 2 (not 1) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Style/Tab: Tab detected. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)